packages/protocol/contracts/libs/LibBlockHeader.sol

```diff
@@ -34,7 +34,15 @@ library LibBlockHeader {
     function hashBlockHeader(
         BlockHeader memory header
     ) internal pure returns (bytes32) {
-        bytes[] memory list;
+        bytes memory rlpHeader = LibRLPWriter.writeList(
+            getBlockHeaderRLPItemsList(header)
+        );
+        return keccak256(rlpHeader);
+    }
+
+    function getBlockHeaderRLPItemsList(
```

Comment:
make it private and rename to `_getBlockHeaderRLPItemsList`?

Reply:
we still need it to be `internal` since Its also used by `LibProving._getInstance`?